pallets/proposals/src/lib.rs

```diff
@@ -198,6 +198,13 @@ pub mod pallet {
             BalanceOf<T>,
             common_types::CurrencyId,
         ),
+        // Project has been updated
+        ProjectUpdated(
```

Comment:
What are we using this data for?

Reply:
This is an event definition that gets triggered at the end of the extrinsinc execution